state/protocol/util/testing.go

```diff
@@ -32,10 +44,25 @@ func RunWithFullProtocolState(t testing.TB, stateRoot *pbadger.StateRoot, f func
 		metrics := metrics.NewNoopCollector()
 		tracer := trace.NewNoopTracer()
 		consumer := events.NewNoop()
-		headers, _, seals, index, payloads, blocks, setups, commits, statuses := util.StorageLayer(t, db)
+		headers, _, seals, index, payloads, blocks, setups, commits, statuses, _ := util.StorageLayer(t, db)
+		state, err := pbadger.Bootstrap(metrics, db, headers, seals, blocks, setups, commits, statuses, stateRoot)
+		require.NoError(t, err)
+		receiptValidator := MockReceiptValidator()
```

Comment:
Comment for reviewers: I am using mocked `ReceiptValidator` in most tests since we are generating fixtures which are invalid if checked by `ReceiptValidator`. There is a separate test suite that tests the validator.